### ISSUE TITLE
Andro

### ISSUE THREAD

**Issue by frank-spec** [android/nowinandroid#1294]
$ repo init -b main -u https://android.googlesource.com/platform/manifest

**Comment by SimonMarquis** [comment]
@frank-spec, I don't think this is related to the nowinandroid repository, please close this issue 🙏
